packages/cms-cli/commands/init.js

```diff
@@ -70,6 +74,30 @@ const AUTH_METHOD_FLOW = {
       process.exit();
     },
   },
+  [USER_TOKEN_AUTH_METHOD.value]: {
+    prompt: async () => {
+      const firstPrompt = await promptUser(USER_TOKEN_FLOW);
+      open(`https://local.hubspot.com/user-token/${firstPrompt.portalId}`);
```

Comment:
I think what we want to do here is leverage a shotlink and then only prompt for the user token. Once we have the user token, we can look up the portal when fetching the access token.

Reply:
I'm not sure that I'm following how we will get them to the user-token-ui with the correct portal without asking them for the portalId.